tests/lib/rules/prefer-object-rule.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @author Brad Zacher <https://github.com/bradzacher>
+ */
+
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/prefer-object-rule');
+const RuleTester = require('eslint').RuleTester;
+
+const ERROR = { messageId: 'preferObject', line: 2, column: 26 };
```

Comment:
why does it also need the `type`?
That's just a superfluous assertion when I'm also including the line/col.
Would also mean I have to redefine the error for one of the tests because it is an arrow function

Reply:
The line/col numbers do not make it immediately clear to the reader what type of node the violation is expected on. Especially when the violation can be reported on different types of nodes, I have found it even more important to be explicit about that. But it's totally up to you until a future lint rule enforces it one way or the other :) https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/issues/59